package.json

```diff
@@ -44,7 +46,7 @@
     "rxjs": "5.0.0-beta.6",
     "select2": "^4.0.3",
     "sortablejs": "^1.4.2",
-    "typescript": "^1.8.10",
+    "typescript": "1.9.0-dev.20160626-1.0",
```

Comment:
I didn't analysed 2.0 yet, so I don't know what impact will this have on the whole project.